src/object/background.cpp

```diff
@@ -459,15 +469,25 @@ std::unordered_map<std::string, std::string> fallback_paths = {
 
 } // namespace
 
-SurfacePtr
+SpritePtr
```

Comment:
Can this method be `const`?

Reply:
In fact, I believe it could be `static`.

Reply:
Okay, so mark this static and the one after that.